### DISCUSSION TITLE
How do I send email notifications to reviewers/assignees when renovatebot creates new MRs through Gitlab?

### DISCUSSION THREAD

**Discussion by ZianZhouBC** [renovatebot/renovate#22263] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

ghcr.io/renovatebot/renovate:35.35.0-slim@sha256:d6fe82bf100dc0759acc8eec0d2f874b281974b70221e86771609bbab68eec08

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Every time a new MR is created on Gitlab by renovatebot, a notification is send to a set group of people mentioned in the configurations. It can either be a gitlab group or a list of usernames. 

### What you tried so far.

`packageRules: [
        {
            "groupName": "foo",
            "reviewers": ["foo"],
            "assignees": ["foo"]
        }
    ],`

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Renovate doesn't send email, and I don't think we want to add that. Platforms like gitlab generally have their own way of notifying people about events

**Reply by ZianZhouBC** [reply]
Is it possible to configure gitlab to do this? It would make sense to be notified whenever a new MR is created

**Reply by ZianZhouBC** [reply]
Also I used the assignees configuration, it didn't seem to do anything. I'm also not sure about the reviewers config, the doc is not too clear about what it actually does. What's the point of using assignees and reviewers if they don't get notified? 

Thank you again for your time and quick response

**Reply by rarkins** [reply, marked as answer]
You'd need to check the gitlab docs to learn what's possible for notifications

**Reply by ZianZhouBC** [reply]
Sounds good, thanks
